ClientApp/KachnaOnline/src/app/home/events-overview/events-overview.component.ts

```diff
@@ -14,33 +14,30 @@ import { endWith } from "rxjs/operators";
 })
 export class EventsOverviewComponent implements OnInit {
```

Comment:
Just a note -- this is going to be in conflict with my calendar PR so you will have to rebase and fix conflict in one of these.